### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

**Review by RabenkoYevhenii — APPROVED** [review]
Looks good

File: cinema/serializers.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField()
```
Comment (Y-Havryliv) [inline cinema/serializers.py:8]: it's better to specify length here as well

**Review by Y-Havryliv — APPROVED** [review]
gj!

File: cinema/serializers.py (hunk 1 above)
Comment (Daniil-Pankieiev) [reply cinema/serializers.py]: Fixed

**Pull request closed without merging**